lesson07/assignment/lesson07/src/router/index.js

```diff
@@ -0,0 +1,44 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import HelloWorld from '@/components/HelloWorld'
+import images from '@/components/images'
+import grid from '@/components/grid'
+import accordion from '@/components/accordion'
+
+Vue.use(VueRouter)
+
+const routes = [
+    {
+      path: '/',
+      name: 'HelloWorld',
+      component: HelloWorld
+    },
+    {
+      path: '/images',
+      name: 'images',
+      component: images
+    },
+    {
+      path: '/grid',
+      name: 'grid',
+      component: grid
+    },
+    {
+      path: '/accordion',
+      name: 'accordion',
+      component: accordion
+    }
+  ]
```

Comment:
Is there a route or context in which the App.vue component shows? If not, we should probably delete the file.